concrete-core/src/specification/engines/lwe_ciphertext_vector_glwe_ciphertext_discarding_packing_keyswitch.rs

```diff
@@ -0,0 +1,95 @@
+use super::engine_error;
+use crate::prelude::{GlweCiphertextEntity, PackingKeyswitchKeyEntity};
+use crate::specification::engines::AbstractEngine;
+
+use crate::specification::entities::LweCiphertextVectorEntity;
+
+engine_error! {
+    LweCiphertextVectorGlweCiphertextDiscardingPackingKeyswitchError for LweCiphertextVectorGlweCiphertextDiscardingPackingKeyswitchEngine @
+    InputLweDimensionMismatch => "The input ciphertext vector and keyswitch key input LWE \
```

Comment:
packing keyswitch key

Reply:
I didn't bother writting "packing" into every single error because the error is called `LweCiphertextVectorGlweCiphertextDiscardingPackingKeyswitchError` so the keyswitching key we are reffering to is obviously a packing KSK. I didn't want to weigh down the error message.